relay/reader/error.go

```diff
@@ -28,3 +28,13 @@ func isIgnorableError(err error) bool {
 	}
 	return false
 }
+
+// isRetryableError checks whether the error is retryable.
+func isRetryableError(err error) bool {
```

Comment:
should we rename it to a better name, add some worker like retry from where

Reply:
let `Reader` itself to retry in https://github.com/pingcap/dm/pull/171/commits/071bc72611e21e93629c04f28fb439ccd9cb4f38. keep its name still be `isRetryableError`.